Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.py

```diff
@@ -114,7 +114,7 @@ def http_request(method, url_suffix, params=None, headers=HEADERS, threshold=THR
         return REPORT_SUCCESS if url_suffix == REPORT_CMD else analysis.json()
     except Exception as e:
         LOG(e)
-        return_error(e.message)
+        return_error(str(e))
```

Comment:
You can remove the log in the line before, It is included in the `return_error` function.